app/controllers/nonsipp/receivetransfer/TransferReceivedMemberListController.scala

```diff
@@ -252,7 +347,48 @@ object TransferReceivedMemberListController {
       buttonText = "site.saveAndContinue",
       details = None,
       onSubmit = controllers.nonsipp.receivetransfer.routes.TransferReceivedMemberListController
-        .onSubmit(srn, page, mode)
+        .onSubmit(srn, page, mode),
+      optViewOnlyDetails = if (mode == ViewOnlyMode) {
```

Comment:
Just for future reference: there is a newly introduced `mode.isViewOnlyMode` val that can be potentially use here.